operations/workplace/people/working-at-mattermost/internal-transfers.md

```diff
@@ -14,6 +23,23 @@ The goal of this discussion is to:
 - Allow HR to confirm potential compensation impact, especially if the transition results in a decrease in compensation.
 - Create a transition plan that balances the needs above with an aligned timeline.
 
+### Process for promotions
+
+**Cadence**
+
+Level changes are reviewed and discussed during every [performance review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50?q=levels). As such, promotions typically happen once a year during the [Winter Performance Review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50/performance-review-timeline-winter-2019). Managers and staff are also encouraged to discuss growth and development opportunities throughout the year. 
+
+**Off-Cycle Cadence**
+
+Off-cycle promotions may also occur based on business need. This is typically determined if a [new job need exists](https://handbook.mattermost.com/contributors/join-us/staff-recruiting#how-to-open-a-new-role). Staff members can express interest in being considered for an [open role](https://mattermost.com/careers/) by reaching out to the Hiring Manager. Similar to transfers, depending on the new role, there may be a need to assess skillset. This will be determined and communicated on a case-by-case basis, and will typically involve components of the internal interview process. 
```

Comment:
```suggestion
Off-cycle promotions may also occur based on business need. This is typically determined if a [new job need exists](https://handbook.mattermost.com/contributors/join-us/staff-recruiting#how-to-open-a-new-role). Staff members can express interest in being considered for an [open role](https://mattermost.com/careers/) by reaching out to the Hiring Manager. Similar to transfers, depending on the new role, there may be a need to assess skillset. This will be determined and communicated on a case-by-case basis, and will typically involve components of the internal interview process.
```